.ci_support/linux_aarch64_.yaml

```diff
@@ -38,6 +38,8 @@ libxml2:
 - '2.11'
 openssl:
 - '3'
+pulseaudio_client:
+- '16.1'
```

Comment:
The rerender is critical for it to pickup this pinning correctly.

Please remember to do this next time.

Thanks!